src/Middleware/Microsoft.AspNetCore.OutputCaching.StackExchangeRedis/src/Microsoft.AspNetCore.OutputCaching.StackExchangeRedis.csproj

```diff
@@ -0,0 +1,29 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <Description>Output cache implementation of Microsoft.AspNetCore.OutputCaching.IOutputCacheStore using Redis.</Description>
+    <TargetFrameworks>$(DefaultNetCoreTargetFramework)</TargetFrameworks>
+    <GenerateDocumentationFile>true</GenerateDocumentationFile>
+    <PackageTags>cache;distributedcache;redis</PackageTags>
+    <ExcludeFromSourceBuild>true</ExcludeFromSourceBuild>
```

Comment:
Does this actually need to be excluded from SourceBuild?

Reply:
This should be removed unless there's a strong reason for it

Reply:
copy pasta from the redis distributed cache lib; removed